### ISSUE TITLE
[BUG]  implementation of Griffin

### ISSUE THREAD

**Issue by ruoyxue** [kyegomez/Griffin#4]
Dear authors，
It seems that the local attention described in the paper is not implemented. Furthermore, I think the initialization of linear layers should be put in the __init__ function, not in the forward function like code below.

        linear_1, linear_2 = nn.Linear(d, d)(x), nn.Linear(d, d)(x)

        linear_1 = nn.Conv1d(
            in_channels=s,
            out_channels=s,
            kernel_size=3,
            padding=1,
        )(linear_1)
